_images/scripts/openturns-ComposedDistribution_for_www.py

```diff
@@ -0,0 +1,90 @@
+"""
```

Comment:
this file has a different naming pattern

Reply:
Ok. Fixed.